### ISSUE TITLE
What is the purpose for adding ce_term in the whole computation?

### ISSUE THREAD

**Issue by JohnD-I** [ratschlab/SVGP-VAE#2]
Hi,
This is an awesome work! I have a quick question about the function "gauss_cross_entropy(mu1, var1, mu2, var2)", which is defined in SVGP-VAE/utils.py file.
This function seems to return the negative cross_entropy and this ce_term is used in later computations. However, this ce_term was not explained in the paper. Could you briefly explain the intuition for introducing this ce_term and what it does/what are the benefits by adding this term? 

Thanks again for the great work!
Best wishes
John

**Comment by metodj** [comment]
Hey John!

The `ce_term` in code corresponds to the cross-entropy between (latent) sparse GP posterior $q_S$ and (rearranged) VAE approximate posterior $\tilde{q}_{\phi}$ in the SVGP-VAE ELBO (see eq. (10) in our paper):

![Screenshot 2022-12-02 at 18 16 21](https://user-images.githubusercontent.com/20437515/205348511-ced23843-d4a5-46a0-a8c2-7138739f0a4e.png)

Hope that answers your question!

Cheers,

Metod

**Comment by JohnD-I** [comment]
Hi Metod!
Is it going to minimize the cross-entropy between (latent) sparse GP posterior $q_S$ and (rearranged) VAE approximate posterior $\tilde{q}_\phi$ during the optimization? Because gauss_cross_entropy function seems to return the negative CE of the two posteriors. Another related question is the benefits for adding this ce_term in optimization, could we simply remove this term in the loss and will the results be very different?
Best wishes
John
